style-dictionary/visual-refresh/borders.ts

```diff
@@ -34,6 +34,7 @@ export const tokens: StyleDictionary.BordersDictionary = {
   borderRadiusItem: '8px',
   borderRadiusInput: '8px',
   borderRadiusPopover: '{borderRadiusInput}',
+  borderRadiusSliderThumb: '50%',
```

Comment:
Do we expect this radius to ever change?

Reply:
Not that I know of, but it seemed like it was good practice to make border radii tokens. Do you see a reason why not?

Reply:
more tokens = bigger CSS bundle

Reply:
Okay, I'll remove the token then!